pkgs/misc/cups/drivers/cnijfilter_2_80/default.nix

```
@@ -17,7 +18,7 @@
   file included in the tarball
 */
 
-gcc13Stdenv.mkDerivation {
+(withCFlags [ "-std=gnu90" ] stdenv).mkDerivation {
```

```suggestion
stdenv.mkDerivation {
```

Just add `env.NIX_CFLAGS_COMPILE = " -std=gnu90"`